content-repo/extra-docs/articles/content-management.md

```diff
@@ -124,48 +183,43 @@ The idea behind the structure is to keep track of all versions in the `productio
 │   │   ├── ...
 ```
 
----
-## The Contents of the Pack
-
-### The `Configuration Setup` Incident type
-#### Custom Fields:
-- `Configuration File Source` - The source of the configuration file.
-- `Custom Packs Source` - The source of the custom packs.
-- `Configuration File Path` - The relative path within the repository to the `xsoar_config.json` file. (Optional)
-- `Branch Name` - The branch from which to fetch the file. Default is the main branch. (Optional)
-
-#### Layout:
-![layout.png](https://raw.githubusercontent.com/demisto/content/master/Packs/ContentManagement/docs-files/layout.png)
-
----
-
-### The `Configuration Setup` Playbook
-This playbook will manage the entire configuration process and needs to run through a `Configuration Setup` incident type.<br />
-It consists of five stages:
-1. Fetching the configuration file and loading its content to the machine.
-2. Downloading and installing the custom packs.
-3. Installing Marketplace packs.
-4. Configuring lists.
-5. Configuring jobs.
-
-![playbook.png](https://raw.githubusercontent.com/demisto/content/master/Packs/ContentManagement/docs-files/playbook.png)
-
----
+## How to Use the Pack
+- [Run the pack manually](#run-the-pack-manually)
+- [Run the pack as a scheduled job](#run-the-pack-as-a-scheduled-job)
+
+### Run the Pack Manually
+1. Navigate to **Incidents**.
+2. Click **New Incident**.
+3. Enter a name for the incident.
+4. From the Type drop down list, choose *Configuration Setup*.
+5. (Optional) From the Configuration File Source field in the Configurations section, select the source of the configuration file for the playbook run. If left empty, the master configuration will be used.
```

Comment:
lets make it clear that by Configuration File we refer to xsoar_config.json file